stdlib/log/logger.lua

```diff
@@ -65,8 +75,12 @@ function Logger.new(mod_name, log_name, debug_mode, options)
             local time_minutes = floor(time_s/60)
             local time_hours = floor(time_minutes/60)
 
+            if type(msg) == 'table' then
```

Comment:
Should we invert this to `if type(msg) ~= 'string'`lua instead here, since serpent will correctly handle all non-string types, and we might not?